datemath/__init__.py

```diff
@@ -1,9 +1,18 @@
-from .helpers import parse, DateMathException
+from __future__ import annotations
 
-def dm(expr, **kwargs):
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing_extensions import Unpack
+    from arrow import Arrow
+
+from .helpers import ParseParams, parse as parse, DateMathException as DateMathException
```

Comment:
` as ` imports without renaming is a typing convention understood by type-checkers to mean that the symbol is purposefully / explicitly re-exported.